src/hooks/get_hk_timezone.ts

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useState } from 'react';
+import moment from 'moment-timezone';
+
+export const useHKT = (publishedAt: string) => {
+  const [time, setTime] = useState(publishedAt);
+  useEffect(() => {
+    const hkt = moment
+      .tz(publishedAt, 'Do MMM YYYY, h:mm a', 'Asia/Hong_Kong')
+      .format('Do MMM YYYY, h:mm a');
+    setTime(hkt);
+  }, [publishedAt]);
+  return time;
+};
```

Comment:
I noticed a small deviation from our code standard in this PR. As per our guidelines, the `useHKT` function should be set as the default export, and the filename should be `useHKT.ts`.

Reply:
changed the file name for `useHKT` hook, and exported the hook as default export. Will work on refactoring the import for all custom hooks on the code base, so for the code base consistency will import this hook from `@hooks` for now. Please let me know if there's any issues! @teamchong